pkg/ansible/runner/runner.go

```diff
@@ -52,56 +52,65 @@ type Runner interface {
 	GetFinalizer() (string, bool)
 }
 
+func ansibleVerbosityString(verbosity int) string {
+	if verbosity > 0 {
```

Comment:
Would not it can be as v, or vv, or vvv, or vvvv? See https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html#cmdoption-ansible-playbook-v

So, wdyt about your solution allow the user to inform it?

Reply:
1. To me it made more sense to make this the integer value of the verbosity level. I don't believe this would be too surprising to Ansible users as that is how [`debug`](https://docs.ansible.com/ansible/latest/modules/debug_module.html#parameters) works. I also wanted to avoid any possible confusion with the operator-sdk's own verbosity.

Reply:
But the `operator-sdk's own verbosity` is in integer. 

So, IMHO it should be done in the same way, which means we could have an env var to setup it  as follows:

```
env:
  - name: ANSIBLE_ VERBOSITY_LEVEL
     value: "vvvv" # it could be  v, or vv, or vvv, or vvvv
```

Also, IHMO we should keep the same values used by ansible instead of creating a new nomenclature and a from/to map for it.